optimus/wdl_tasks/StarAlignFastqSingleEnd.wdl

```diff
@@ -0,0 +1,39 @@
+
+task StarAlignFastqSingleEnd {
+  File fastq_input
+  File gztar_star_reference
+
+  # estimate input requirements for star using UBams:
+  # multiply input size by 2.2 to account for output bam file + 20% overhead, add size of reference.
+  Int input_size = ceil(size(fastq_input, "G"))
+  Int reference_size = ceil(size(gztar_star_reference, "G"))
+  Int estimated_disk_required =  ceil(input_size * 2.2 + reference_size)
+
+  command {
+    # prepare reference
+    mkdir genome_reference && tar -xzf ${gztar_star_reference} -C genome_reference --strip-components 1
```

Comment:
If you drop the `z` flag it should work with both compressed and uncompressed tarballs.

Reply:
I didn't know that. Thank you for the suggestion.

... in what circumstance do you need to pass `-j` or `-z` to `tar -xf`?

Reply:
I think if your version of tar is more than 13 years old
https://www.gnu.org/software/tar/